pkg/userdata/helper/kubelet.go

```diff
@@ -295,6 +295,28 @@ func kubeletConfiguration(log *zap.SugaredLogger, clusterDomain string, clusterD
 		}
 	}
 
+	if imgGcLowThreshold, ok := kubeletConfigs[common.ImageGCLowThresholdPercentKubeletConfig]; ok {
+		val, err := strconv.ParseInt(imgGcLowThreshold, 10, 32)
+		if err != nil || val < 0 || val > 100 {
+			// Instead of breaking the workflow, just print a warning and skip the configuration
+			log.Info("Skipping invalid ImageGCLowThresholdPercent value for Kubelet configuration", "value", imgGcLowThreshold)
```

Comment:
Is it possible to also add an event to the machinedeployment?
That way the user also has the possibility to see that something was wrongly configured

Reply:
That's a good point, but it would be difficult because this code runs in the userdata plugin binary, which doesn't even have easy K8s API access AFAIK. I just copied the behaviour from the other settings, which also just log an error if something is wrong with the annotation value.
So if we want to change this, it should be a separate ticket.